plugins/modules/zos_replace.py

```diff
@@ -81,7 +97,10 @@
   literal:
```

Comment:
Since there are pending reviews, I can not multi select the entire section of text for `literal` so I am going to offer my rewrite that also updates the URL so there is no version embedded, also makes the latest Ansible markup `O(...)` and other things like `B(bold)`. This is for the entire `literal` section of text.

```
  - If specified, it enables the module to interpret options O(after), O(before) and O(regex) as a literal rather than a regular expression.
  - Option O(literal) uses any combination of V(after), V(before) and V(regexp).
  - To interpret one option as a literal, use O(literal=regexp), O(literal=after) or O(literal=before).
  - To interpret multiple options as a literal, use a list such as C(['after', 'before']) or C(['regex', 'after', 'before'])
```